### PR TITLE
Add blog post on State of Trino talk at Trino Summit

### PR DESCRIPTION
**Pull request by colebow**
A pass at a quick recap of the State of Trino talk that Martin gave.

### CODE REVIEW

File: _posts/2022-11-16-trino-summit-keynote-recap.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: "Trino Summit recap: keynote and the state of Trino"
```
Comment (bitsondatadev) [inline _posts/2022-11-16-trino-summit-keynote-recap.md:3]:
```suggestion
title: "Trino Summit 2022 recap: The state of Trino"
```

File: _posts/2022-11-16-trino-summit-keynote-recap.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+Welcome to the first in a series of posts recapping everything interesting that
+went down at the 2022 Trino Summit! There is a post being published each day, so
+stay tuned for more as the next couple week go on. To kick the summit off, we 
+heard from Trino co-creators Martin Traverso, Dain Sundstrom, and David
+Phillips. Martin gave a talk on the state of Trino and project plans for 2023,
+then opened the floor to questions from the community. Check out the recording
+of the talk below, or read on if you're only interested in the highlights.
+
+|| -- embed YouTube video here -- ||
```
Comment (bitsondatadev) [inline _posts/2022-11-16-trino-summit-keynote-recap.md:17]:
```suggestion
{% youtube mUq_h3oArp4 %}
```

File: _posts/2022-11-16-trino-summit-keynote-recap.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+image: /assets/blog/trino-summit-2022/keynote-header.jpg
+---
+
+Welcome to the first in a series of posts recapping everything interesting that
+went down at the 2022 Trino Summit! There is a post being published each day, so
+stay tuned for more as the next couple week go on. To kick the summit off, we 
+heard from Trino co-creators Martin Traverso, Dain Sundstrom, and David
+Phillips. Martin gave a talk on the state of Trino and project plans for 2023,
+then opened the floor to questions from the community. Check out the recording
+of the talk below, or read on if you're only interested in the highlights.
```
Comment (bitsondatadev) [inline _posts/2022-11-16-trino-summit-keynote-recap.md:15]:
```suggestion
Welcome to the first in a series of posts recapping everything interesting that
went down at the 2022 Trino Summit! There is a post being published for each
session, so stay tuned for more in the next couple of weeks. To kick the summit
off, we heard from Trino co-creators Martin Traverso, Dain Sundstrom, and
David Phillips. Martin gave a talk on the state of Trino and project plans for
2023, then opened the floor to questions from the community. Check out the
recording of the talk below, or read on if you're only interested in the highlights.
```

**Review by bitsondatadev — CHANGES_REQUESTED** [review]

**Comment by bitsondatadev** [comment]
Also, please rename the file to `_posts/2022-11-16-trino-summit-2022-keynote-recap.md`

**Review by bitsondatadev — CHANGES_REQUESTED** [review]
A few more updates based on us wanting to use the images from the photographer in the header. Also we need a couple headers to break things up a bit.

File: _posts/2022-11-22-trino-summit-2022-state-of-trino-keynote-recap.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+Welcome to the first in a series of posts recapping everything interesting that
+went down at the 2022 Trino Summit! There is a post being published for each
+session, so stay tuned for more in the next couple of weeks. To kick the summit
+off, we heard from Trino co-creators Martin Traverso, Dain Sundstrom, and David
+Phillips. Martin gave a talk on the state of Trino and project plans for 2023,
+then opened the floor to questions from the community. Check out the recording
+of the talk below, or read on if you're only interested in the highlights.
+
+{% youtube mUq_h3oArp4 %}
```
Comment (mosabua) [inline _posts/2022-11-22-trino-summit-2022-state-of-trino-keynote-recap.md:17]: Video should go below the fold

File: _posts/2022-11-22-trino-summit-2022-state-of-trino-keynote-recap.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: "Trino Summit 2022 recap: The state of Trino"
+author: "Martin Traverso, Dain Sundstrom, David Phillips, Cole Bowden"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/keynote-header.png
+---
+
+Welcome to the first in a series of posts recapping everything interesting that
+went down at the 2022 Trino Summit! There is a post being published for each
```
Comment (mosabua) [inline _posts/2022-11-22-trino-summit-2022-state-of-trino-keynote-recap.md:10]: I dont think we should explain what we are doing .. just remove the first two sentences and replace it with a link to the post as part of the next sentence 


```
To kick off [Trino Summit 2022](link), we heard
```

**Review by mosabua — COMMENTED** [review]
Please update and add link to slide deck as well. Then let me know when its ready again.

File: _posts/2022-11-22-trino-summit-2022-state-of-trino-keynote-recap.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+then opened the floor to questions from the community. Check out the recording
+of the talk below, or read on if you're only interested in the highlights.
+
+<!--more-->
+
+## The State of Trino talk
+
+{% youtube mUq_h3oArp4 %}
+
+[Download a PDF of the slides here.](https://www.starburst.io/wp-content/uploads/2022/11/State-of-Trino-Nov-2022.pdf)
```
Comment (mosabua) [inline _posts/2022-11-22-trino-summit-2022-state-of-trino-keynote-recap.md:21]:
```suggestion
[Download a PDF of the slides](https://www.starburst.io/wp-content/uploads/2022/11/State-of-Trino-Nov-2022.pdf)
```

File: _posts/2022-11-22-trino-summit-2022-state-of-trino-keynote-recap.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+## Recapping
+
+So what *has* happened in Trino over the last year?
+
+* We celebrated Trino's 10th birthday!
```
Comment (mosabua) [inline _posts/2022-11-22-trino-summit-2022-state-of-trino-keynote-recap.md:27]: link to blog post

File: _posts/2022-11-22-trino-summit-2022-state-of-trino-keynote-recap.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+* It was the busiest year in project history, with 600+ contributors, 4000+
+  commits, and near-weekly releases.
+* Tons of new features were added, including `MERGE`, JSON functions, table
+  functions, fault-tolerant execution (look forward to a lot of talking about it
```
Comment (mosabua) [inline _posts/2022-11-22-trino-summit-2022-state-of-trino-keynote-recap.md:31]: get rid of this promise for a future post

File: _posts/2022-11-22-trino-summit-2022-state-of-trino-keynote-recap.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+  getting code merged into Trino.
+* New connectors, `CREATE/DROP CATALOG`, query tracing, and more!
+
+There were also tons of great questions asked by live and online attendees
+answered by Dain, David, and Martin, so if you want to hear more, take a listen
+to the full talk!
+
+## Share this session
+
+If you thought this talk was interesting, please consider sharing this on
```
Comment (mosabua) [inline _posts/2022-11-22-trino-summit-2022-state-of-trino-keynote-recap.md:50]: drop "please"

**Review by mosabua — CHANGES_REQUESTED** [review]
Very close to ready ;-)

File: _posts/2022-11-22-trino-summit-2022-state-of-trino-keynote-recap.md (hunk 6 above)
Comment (colebow) [reply _posts/2022-11-22-trino-summit-2022-state-of-trino-keynote-recap.md]: I've made it a button and much fancier.

File: _posts/2022-11-22-trino-summit-2022-state-of-trino-keynote-recap.md (hunk 8 above)
Comment (colebow) [reply _posts/2022-11-22-trino-summit-2022-state-of-trino-keynote-recap.md:32]: I like it! And we're going to talk about it, I think it's ok to make a promise if we're 100% going to follow-through on it.

**Review by bitsondatadev — APPROVED** [review]
Looks good to me. I'm also good with saying we're gonna talk about FTE more. Other subjects I may not be so sure but that's something we're gonna market a lot more anyways. So it's safe.

Good work!

File: _posts/2022-11-22-trino-summit-2022-state-of-trino-keynote-recap.md (hunk 8 above)
Comment (mosabua) [reply _posts/2022-11-22-trino-summit-2022-state-of-trino-keynote-recap.md:32]: we are not .. just look at the blog post we had in the pipeline for a few months now

**Review by bitsondatadev — CHANGES_REQUESTED** [review]
Actually @colebow I just remembered, please add a link from the main recap post to this post please. 

See examples in the Apple and Zillow post.

**Review by mosabua — APPROVED** [review]
Good to go now from my view.

**Review by bitsondatadev — APPROVED** [review]

**Pull request merged**